streams/buffer.ts

```diff
@@ -375,9 +375,6 @@ export class Buffer {
   }
 
   #reslice(len: number) {
-    if (len > this.#buf.buffer.byteLength) {
```

Comment:
Is this condition unsatisfiable? If so, how?

Reply:
It's guarded by the condition before calling #reslice(). It seems not possible to make `len` exceed `this.#buf.buffer.byteLength`